lib/rules/mustache-curly-spacing.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @fileoverview Enforce spacing on the style of mustache curly brackets.
+ * @author Armano
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Enforce spacing on the style of mustache curly brackets.',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: 'whitespace',
+    schema: [
+      {
+        enum: ['always', 'never']
+      }
+    ]
+  },
+
+  create (context) {
+    const options = context.options[0]
+    const optSpaces = options !== 'never'
+    const template = context.parserServices.getTemplateBodyTokenStore && context.parserServices.getTemplateBodyTokenStore()
+
+    // ----------------------------------------------------------------------
+    // Helpers
+    // ----------------------------------------------------------------------
+
+    function checkTokens (leftToken, rightToken) {
+      if (leftToken.loc.end.line === rightToken.loc.start.line) {
+        const spaces = rightToken.loc.start.column - leftToken.loc.end.column
+        if (optSpaces === (spaces === 0)) {
```

Comment:
Don't you think this expression is hard to read?
Can you please keep it simple stupid?
```
const noSpacesFound = spaces === 0
if (optSpaces && noSpacesFound) {
```